libs/webb-ui-components/src/components/Table/Table.tsx

```diff
@@ -22,53 +24,55 @@ const TableComp = <T extends RowData>(
 ) => {
   return (
     <div {...props} ref={ref}>
-      <table className="w-full border-collapse table-auto">
-        <thead>
-          {table.getHeaderGroups().map((headerGroup) => (
-            <tr key={headerGroup.id}>
-              {headerGroup.headers.map((header) => (
-                <THeader className={thClassName} key={header.id}>
-                  {header.isPlaceholder
-                    ? null
-                    : flexRender(
-                        header.column.columnDef.header,
-                        header.getContext()
-                      )}
-                </THeader>
-              ))}
-            </tr>
-          ))}
-        </thead>
-        <tbody>
-          {table.getRowModel().rows.map((row) => (
-            <tr key={row.id} className="group/tr">
-              {row.getVisibleCells().map((cell) => (
-                <TData className={tdClassName} key={cell.id}>
-                  {flexRender(cell.column.columnDef.cell, cell.getContext())}
-                </TData>
-              ))}
-            </tr>
-          ))}
-        </tbody>
-        {isDisplayFooter && (
-          <tfoot>
-            {table.getFooterGroups().map((footerGroup) => (
-              <tr key={footerGroup.id}>
-                {footerGroup.headers.map((header) => (
-                  <THeader key={header.id}>
+      <div className={tableClassName}>
```

Comment:
Why not try using the tableClassName directly with the <table></table> tag instead of a <div></div>? This could be a better approach. Plus, you can take advantage of twMerge to combine the current styling class with the outer class. It's quite handy!